docker/gcp_pudl_etl.sh

```diff
@@ -24,25 +24,30 @@ function run_pudl_etl() {
     && pudl_setup \
     && ferc_to_sqlite \
         --loglevel=DEBUG \
+        --logfile=${PUDL_OUTPUT}/ferc-to-sqlite.log \
         --gcs-cache-path=gs://internal-zenodo-cache.catalyst.coop \
         --workers=8 \
         $PUDL_SETTINGS_YML \
     && pudl_etl \
         --loglevel DEBUG \
+        --logfile=${PUDL_OUTPUT}/pudl-etl.log \
         --gcs-cache-path gs://internal-zenodo-cache.catalyst.coop \
         --partition-epacems \
         $PUDL_SETTINGS_YML \
     && pytest \
         --gcs-cache-path=gs://internal-zenodo-cache.catalyst.coop \
         --etl-settings=$PUDL_SETTINGS_YML \
+        --log-file=${PUDL_OUTPUT}/pytest.log \
         --live-dbs test
 }
 
 function shutdown_vm() {
     # Copy the outputs to the GCS bucket
     gsutil -m cp -r $PUDL_OUTPUT "gs://nightly-build-outputs.catalyst.coop/$ACTION_SHA-$GITHUB_REF"
 
-    upload_file_to_slack "${PUDL_OUTPUT}/pudl-etl.log" "Logs for $ACTION_SHA-$GITHUB_REF:"
+    upload_file_to_slack "${PUDL_OUTPUT}/ferc-to-sqlite.log" "ferc_to_sqlite logs for $ACTION_SHA-$GITHUB_REF:"
```

Comment:
Do we prefer looking at raw logs vs. in Cloud Logging? Seems like the extra filtering tools from Cloud Logging might be handy, and the fact that they stream in vs. being uploaded at the end is nice too.